.github/workflows/acceptance.yml

```diff
@@ -11,7 +11,7 @@ on:
     branches:
       - master
       - releases/**
-  pull_request:
+  pull_request_target:
```

Comment:
I'd add a comment here to guard against future copypaste that we do this instead of pull_request because visual snapshots needs it to work for forked PRs.

Reply:
Or it might be fine to just replace with pull_request_target across the board?

Reply:
yeah I'd prefer to use `pull_request_target` only when necessary because its behavior can be a bit unexpected.